### ISSUE TITLE
Mass Mindflay is confusing, and doesn't seem to work reliably

### ISSUE THREAD

**Issue by MoonlitJolteon** [walksanatora/HexTweaks#9]
Documentation is vague, cant quite tell if I'm doing this right, and if I am, can't tell if it's working. Upon completion of the circle it neither mishaps nor spawns a slipway. Also does not appear to effect the villagers in the circle.

Having tested again it now crashes when it completes, crashlog: https://mclo.gs/InKYptn

A structure block file has also been attached, in case you wish to copy the exact spell circle and villagers into your world for testing:
[spell-circle.zip](https://github.com/walksanatora/HexTweaks/files/13484398/spell-circle.zip)

**Comment by X66Herobrine66X** [comment]
Agreed, the documentation sucks ass and I have no idea what I'm doing. So confusing.

**Comment by MoonlitJolteon** [comment]
It appears the current version of the source code does not have this crash. I built from source and found that the spell worked perfectly.
